docs/api/cluster/elbow.rst

```diff
@@ -8,21 +8,23 @@ The ``KElbowVisualizer`` implements the "elbow" method to help data scientists s
 To demonstrate, in the following example the ``KElbowVisualizer`` fits the ``KMeans`` model for a range of :math:`K` values from 4 to 11 on a sample two-dimensional dataset with 8 random clusters of points. When the model is fit with 8 clusters, we can see an "elbow" in the graph, which in this case we know to be the optimal number.
 
 .. plot::
+    :context: close-figs
+    :alt: KElbowVisualizer on synthetic dataset with 8 random clusters
 
     from sklearn.cluster import KMeans
     from sklearn.datasets import make_blobs
 
     from yellowbrick.cluster import KElbowVisualizer
 
-    # Create synthetic dataset with 8 random clusters
-    X, y = make_blobs(centers=8, n_features=12, shuffle=True, random_state=42)
+    # Generate synthetic dataset with 8 random clusters
+    X, y = make_blobs(n_samples=1000, n_features=12, centers=8, random_state=42)
```

Comment:
I removed `shuffle=True` since it is `True` by default.

In addition, I added `n_samples=1000` since that is what it was set to in the original script used to generate the image, and if not included, `n_samples` is only equal to `100` by default.